### PR TITLE
High sierra libv8 fix

### PR DESCRIPTION
**Pull request by lyrahall14**
Here is the way that I got Arooo to run on my MacOSX High Sierra environment. I am not sure of the best way to apply this to other users but I will just document what I did. The original error message I got when I ran the 'bundle' command is in pull request #284 . As I understand it, the libv8 dependency would not build because there is not a binary on github (?) that is compatible with High Sierra - as per this issue (https://github.com/cowboyd/libv8/issues/256).

1. I ran the command `gem install therubyracer -v 0.12.2`. This successfully installed a version of therubyracer from rubygems.org that included the libv8 binary (libv8-3.16.14.19-x86_64-darwin-17.gem) that is High Sierra compatible.

2. Next I had to point my app at the local copy of therubyracer so it would not keep trying to fetch the version of libv8 that did not work. I followed the advice of this post (https://rossta.net/blog/how-to-specify-local-ruby-gems-in-your-gemfile.html). I ran the command `bundle config --local local.therubyracer `~/.gem/ruby/2.3.5/gems/therubyracer-0.12.2` .

3. I then added the line `gem 'therubyracer', github:"https://github.com/lyrahall14/therubyracer-high-sierra", branch: "master"` to the Gemfile. To make this reference, I had to make the directory ~/.gem/ruby/2.3.5/gems/therubyracer-0.12.2 a git repository and push it to github. Note: I would have used the "real" repository of therubyracer to link to instead, but the whole issue was that it wasn't on github in the first place, I think? I may be understanding this wrong.

4. I ran `bundle` again and it totally worked! Super sweet green text.

So is this fix repeatable for all High Sierra users? Is it the best option? I have no idea, please let me know your thoughts.

### CODE REVIEW

**Comment by lyrahall14** [comment]
Hi @arbonap thank you for reviewing my PR! Your fix makes very good sense and seems much more workable than what I did. I'm pretty new to Ruby so thank you for the detailed explanation 😄 

I'll try out your fix on my setup and comment the results here!

**Comment by lyrahall14** [comment]
Update: with a minor change, it totally worked!

Here's what I ran:

`gem uninstall -a libv8
gem uninstall -a therubyracer
gem install libv8 -v '3.16.14.19' -- --with-system-v8
gem install therubyracer -v '0.12.2' -- --with-v8-dir=$(brew --prefix v8-315)`

The minor change was switching the version of libv8 in the third line to '... 19' instead of '... 13' since that represents the x86_64-darwin-17 build for High Sierra.

Now I'm seeing that sweet sweet homepage!

File: Gemfile.lock [hunk 1]
```diff
@@ -163,7 +172,7 @@ GEM
     kgio (2.10.0)
     launchy (2.4.3)
       addressable (~> 2.3)
-    libv8 (3.16.14.13)
+    libv8 (3.16.14.19)
```
Comment (gitsteph) [inline Gemfile.lock]: @lyrahall14 When I set this up on my machine (running High Sierra), all I needed to do to get it working is to make this lone change (swapping 13 => 19) and nothing else.  Successfully `bundle install` after making the lockfile change.  Could you try removing the other changes, merging in the latest changes from `master`, and seeing if it'll pass the tests?

Comment (Imperiopolis) [reply Gemfile.lock]: The PR I just merged (#297) contains the above change and for me also fixed this issue under High Sierra.

Comment (gitsteph) [reply Gemfile.lock]: 👍 thanks! closing this issue now then

**Pull request closed without merging**